app/src/main/java/com/blackboxteam/app/blackbox/LaunchScreenActivity.java

```diff
@@ -0,0 +1,31 @@
+package com.blackboxteam.app.blackbox;
+
+import android.content.Intent;
+import android.support.v7.app.AppCompatActivity;
+import android.os.Bundle;
+import android.view.View;
+import android.widget.Button;
+
+
+public class LaunchScreenActivity extends AppCompatActivity implements View.OnClickListener {
+
+    Button nextScreen;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_launch_screen);
+
+        nextScreen = (Button) findViewById(R.id.button_secure);
```

Comment:
This works fine but can quickly become cumbersome when you have a lot of views. You'll have to `findViewById(...)` for each one of them. Checkout Butter Knife (http://jakewharton.github.io/butterknife/), it's a library that automatically binds your views. Or you can also use the native databinding features of the Android framework (https://developer.android.com/topic/libraries/data-binding/index.html).